layout-webapp/src/main/webapp/vue-app/common-layout-components/components/site-navigation/SiteNavigationElementDrawer.vue

```diff
@@ -86,7 +86,7 @@ export default {
       return this.$t('siteNavigation.addElementDrawer.title');
     },
     disabled() {
-      return !this.isValidForm || !this.pageTemplate || false;
+      return !this.isValidForm || !this.pageTemplate  || false;
```

Comment:
Extra space, please run `npm run lint` as well to automatically another spacing problem.
 
```suggestion
      return !this.isValidForm || !this.pageTemplate || false;
```

Reply:
done